docs/howto/test.md

```diff
@@ -0,0 +1,18 @@
+# How to test
+
+## Test helpers
+
+ActiveModelSerializers provides a `assert_serializer` method to be used on your controller tests to
+assert that a specific serializer was used.
+
+```ruby
+class PostsControllerTest < ActionController::TestCase
+  test "should render post serializer" do
+    get :index
+    assert_serializer "PostSerializer"
+  end
+end
+```
+
```

Comment:
@trek Yeah, I intend to add that if someone else doesn't... maybe it would be good to make an issue? I've been meaning to write it up as code in https://github.com/rails-api/active_model_serializers/pull/967 but it hasn't 'happened' yet.

In the meantime, I guess people can post links here :)